oracle-price-monitor/src/autotask.js

```diff
@@ -0,0 +1,270 @@
+// ORACLE PRICE ALERT - note there's never been an alert to this agent yet
+// agent id - 0x8a6e2ae3b279c561f7ba64e5011313df19e08b1f398d59d94dfbd4148e6cafce
+
+const axios = require('axios');
+const ethers = require('ethers');
+const needle = require('needle');
```

Comment:
Is `needle` available in the Defender environment? (I assume that it is, or the autotask would perpetually fail).